### ISSUE TITLE
1.12.2 no textures for mobs

### ISSUE THREAD

**Issue by CogdeArtho** [DrZhark/mocreatures#149]
I just got this mod for my own mob pack to challenge myself, and when I used this mod I don't see any textures from the mobs, i can hear sounds, and even kill mobs also get loot but I can't see them at all!

**Comment by anthuoon** [comment]
Yeah, it also happened to me. I can hear bird sounds but i can't see where the birds are. Also, i got attacked by something, then i went to the ocean near me and i see bubbles on the water, i was like "What!?" and i toggle the hitboxes by pressing "F3+B" and i see a hitbox and i had Hwyla installed so i hover my crosshair to that hitbox and it was a lion. Please patch this, i don't want to turn my hitbox on all the time...

**Comment by Fris-Arhaf** [comment]
Yeah, it also happened to me

**Comment by lilwilisjr** [comment]
same problem here pls fix

literally useless since you cant even see anything

**Comment by Null-151** [comment]
Same problem sadly. Mobs spawn naturally and through eggs too, but they're just transparent

**Comment by lilwilisjr** [comment]
> Same problem sadly. Mobs spawn naturally and through eggs too, but they're just transparent

i redownloaded the files and put them in, and it started working again.

make sure the mod file name is like this

DrZharks MoCreatures Mod-12.0.5.jar

**Comment by Null-151** [comment]
Can confirm this worked @lilwilisjr , this issue should be closed

**Comment by Imogenary** [comment]
Am not sure why this happened to you, perhaps you accidentally installed it wrong? I installed it through MultiMC and it works fine, doing it through the normal way (when importing it into your normal MC launcher, it is a lot harder and things are more likely to mess up)

**Comment by Termanty** [comment]
I had also pluses like this DrZharks+MoCreatures+Mod-12.0.5 in the file name and I did not have textures.  @lilwilisjr advise to change file name to 
DrZharks MoCreatures Mod-12.0.5.jar
removed problem. Thanks!!!

**Comment by ZovaZey** [comment]
Are you writing it with the spaces in between? Or like DrZharksMoCreaturesMod-12.0.5.jar ?

**Comment by Termanty** [comment]
DrZharks MoCreatures Mod-12.0.5.jar
Spaces between the words
